examples/aerospike-cluster-values.yaml

```
@@ -0,0 +1,103 @@
+## Aerospike cluster size
+replicas: 3
+
+## Aerospike Database Docker image
+image:
+  repository: aerospike/aerospike-server-enterprise
+  tag: 7.1.0.0
```

```suggestion
  tag: 7.2.0.3
```


use the latest image

---

done